src/shared/bootspec.c

```diff
@@ -1263,13 +1256,7 @@ int boot_config_augment_from_loader(
 }
 
 static int boot_entry_file_check(const char *root, const char *p) {
```

Comment:
You should remove (inline) this function at this point…

Reply:
we don't do `inline` for local `static` functions. The compiler should really be smart enough to inline local functions on its own, and shouldn't need the `inline` hint for that.

`inline` is useful in headers, because it allows us to define function bodies in headers without this becoming a linking issue later.

Reply:
I don't mean add inline keyword. There's only one caller for this, and this function has a single function call now. Just manually inline this and get rid of this function.